fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4312,7 +4417,9 @@ class FileBackupAgentImpl {
 
 	//used for correctness only, locks the database before discontinuing the backup and that same lock is then used while doing the restore.
 	//the tagname of the backup must be the same as the restore.
-	ACTOR static Future<Version> atomicRestore(FileBackupAgent* backupAgent, Database cx, Key tagName, Standalone<VectorRef<KeyRangeRef>> ranges, Key addPrefix, Key removePrefix) {
+	ACTOR static Future<Version> atomicRestore(FileBackupAgent* backupAgent, Database cx, Key tagName,
+	                                           Standalone<VectorRef<KeyRangeRef>> ranges, Key addPrefix,
+	                                           Key removePrefix, bool fastRestore = false) {
```

Comment:
Seems no need to give a default value, since it's a static function. It's preferred not to use default parameters (https://google.github.io/styleguide/cppguide.html#Default_Arguments).

```suggestion
	                                           Key removePrefix, bool fastRestore) {
```